README.md

```diff
@@ -265,7 +275,10 @@ Bal:NGN50,000.00
 ```
 
 You can inject this fake SMS into the emulated phone by following these steps.
-It is advisable that you pre-populate the emulated phone with the SMS _before_ running the sample app.
+
+> Once the app gets the permissions form the user it will instantly start sending existing SMS messages to the Pngme system. This results in messages being seen way sooner than SMS received after the app was installed.
```

Comment:
maybe replace 'way' with 'much'?  'Way' sounds very informal..

Reply:
I did not realise about that detail. Thanks!